src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,55 @@
+package duke.tasks;
+import java.time.*;
+import java.time.format.*;
+
+public class Event extends Task {
+    private LocalDateTime start;
+    private LocalDateTime end;
+
+    //Constructor
+    public Event (String name, LocalDateTime start, LocalDateTime end) {
+        super(name);
+        this.start = start;
+        this.end = end;
+        this.info = "E,0," + name + "," + start.getYear() + "," + start.getMonthValue() + "," + start.getDayOfMonth()
+                + "," + start.getHour() + "," + start.getMinute() + "," + end.getYear() + ","
+                + end.getMonthValue() + "," + end.getDayOfMonth() + "," + end.getHour() + "," + end.getMinute();
+    }
+
+    //Accessors
+    public LocalDateTime getStartTime() {
+        return this.start;
+    }
+
+    public LocalDateTime getEndTime() {
+        return this.end;
+    }
+
+    @Override
+    public void mark() {
+        this.isDone = true;
+        this.info = "E,1," + name + "," + start.getYear() + "," + start.getMonthValue() + "," + start.getDayOfMonth()
+                + "," + start.getHour() + "," + start.getMinute() + "," + end.getYear() + ","
```

Comment:
It is a bit nitpicking but you might want to make the format consistent: move the first `","` of the second line to the first line or make the third line start with a `","`.